samples/snippets/create_lite_pubsub_export_subscription_example.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python
+
+# Copyright 2022 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This application demonstrates how to create a subscription that writes
+messages to a Pub/Sub topic. For more information, see documentation at
+https://cloud.google.com/pubsub/lite/docs/export-subscriptions.
+"""
+
+import argparse
+
+
+def create_lite_pubsub_export_subscription(
+    project_number,
+    cloud_region,
+    zone_id,
+    topic_id,
+    subscription_id,
+    pubsub_topic_id,
+    regional,
+    target_location,
+):
+    # [START pubsublite_create_pubsub_export_subscription]
+    from google.api_core.exceptions import AlreadyExists
+    from google.cloud.pubsub_v1 import PublisherClient
+    from google.cloud.pubsublite import AdminClient, Subscription, ExportConfig
+    from google.cloud.pubsublite.types import (
+        CloudRegion,
+        CloudZone,
+        SubscriptionPath,
+        TopicPath,
+    )
+
+    # TODO(developer):
+    # project_number = 1122334455
+    # cloud_region = "us-central1"
+    # zone_id = "a"
+    # topic_id = "your-topic-id"
+    # subscription_id = "your-subscription-id"
+    # pubsub_topic_id = "destination-topic-id"
+    # regional = True
+    # target_location = BacklogLocation.BEGINNING
```

Comment:
We shouldn't include commented out code, and should use minimal arguments: https://googlecloudplatform.github.io/samples-style-guide/#minimal-arguments

For suggesting default values for users, python has two options:
1. Default arguments:
```python
def example_snippet(project_number="1122334455", cloud_region="us-central1"):
```

2. Use an "easy-run" wrapper function:
```python
def run_snippet():
    project_number = "1122334455"
    cloud_region = "us-central1"
   example_snippet(project_number, region)   

def example_snippet(project_number, region):
```

Reply:
Thanks. I've opted for 1) default arguments, as they need to be set by tests. Also for resemblance to existing samples.